spec/buffer_spec.moon

```diff
@@ -1,6 +1,8 @@
 import Buffer, config, signal from howl
 import File from howl.io
 import with_tmpfile from File
+ffi = require 'ffi'
+ffi.cdef [[ void g_usleep(unsigned long ms); ]]
```

Comment:
Let's move the cdef into the ordinary `glib.moon` cdef file inside of ljglibs instead.